app/gateway/reserved-entity-names.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Reserved entity names"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+works_on:
+    - on-prem
+    - konnect
+
+tags:
+    - entities
+
+min_version:
+  gateway: '3.4'
+
+description: See which {{site.base_gateway}} entity names are reserved.
+
+related_resources:
+  - text: Gateway entities
+    url: /gateway/entities/
+---
+
+The {{site.base_gateway}} maintains a list of reserved entity names which can't be used when naming [Workspaces](/gateway/entities/workspace/) or [other entities](/gateway/entities/).
```

Comment:
```suggestion
{{site.base_gateway}} maintains a list of reserved entity names which can't be used when naming [Gateway entities](/gateway/entities/).
```
Doesn't seem like there's any need to specifically call out workspaces here, especially since the example uses workspaces.